third_party/hamcrest-core.jar

```diff
@@ -1 +0,0 @@
-hamcrest-core-1.3.jar
```

Comment:
Are you sure these can be deleted?  There may be some environments where they are still needed.